feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -214,12 +230,14 @@ func findMatchedComponents(t *testing.T, dut *ondatra.DUTDevice, r *regexp.Regex
 func findStandbyRP(t *testing.T, dut *ondatra.DUTDevice, supervisors []string) (string, string) {
 	var activeRP, standbyRP string
```

Comment:
(no action required on this PR) If these remain empty strings it means (unless a supervisor is the empty string) that it was not found. Should this be documented as a doc comment for the function? If this is not expected should an error be returned? What happens if there are multiple supervisors with the same role?

Considering this issue was here before I think this is best addressed in a separate PR.

Reply:
Ack. We fail the test if the number of controller is < 2.

Will add the check to expect one active and one standby controller in a separate PR.